Shop/src/Appliance.java

```diff
@@ -0,0 +1,21 @@
+public class Appliance extends ElectronicProduct {
+    public Appliance() {
```

Comment:
E destul de riscant sa pui niste valori default, daca nu sunt specificate in cerinta.